nidn/fdtd_integration/compute_spectrum_fdtd.py

```diff
@@ -172,26 +172,56 @@ def _average_along_detector(signal):
     return avg
 
 
-def _summed_thickness_times_permittivity(thicknesses, permittivity):
+def _summed_thickness_times_sqrt_permittivity(thicknesses, permittivity):
+    """
+    Helper function to calculate the sum of the product of the thickness and
+    the square root of the permittivity for each layer in a material stack.
+
+    Args:
+        thicknesses (tensor): tensor with the thickness of each layer of the material stack
+        permittivity (tensor): tensor with the real relative permittivity for each layer of the material stack
```

Comment:
```suggestion
        permittivity (tensor): tensor with the relative permittivity for each layer of the material stack
```
(contains both at this point I think but you only use real part)

Reply:
Yeah true